e2e_test/ddl/show.slt

```diff
@@ -79,21 +79,21 @@ statement ok
 drop table t3;
 
 query TT
-describe pg_matviews_info;
+describe pg_matviews;
```

Comment:
Shall we also add some e2e test on pg_attribute?

Reply:
Ok, I will update tests in commits later.